controllers/jobs.js

```diff
@@ -163,6 +203,41 @@ exports.deleteJob = asyncHandler(async (req, res, next) => {
     }
   }
 
+  const rows = await db.one(
+    'UPDATE jobs SET deleted_on=to_timestamp($2 / 1000.0) WHERE job_id = $1 RETURNING *',
+    [req.params.id, Date.now()]
+  );
+
+  res.status(200).json({
+    success: true,
+    data: rows,
+  });
+});
+
+/**
+ * @desc    Delete single job
+ * @route   DELETE /api/jobs/:id
+ * @access  Admin
+ */
+exports.deleteJob = asyncHandler(async (req, res, next) => {
+  // check if job exists
+  const job = await db.oneOrNone(
+    'SELECT * FROM jobs WHERE job_id = $1',
+    req.params.id
+  );
+
+  // return bad request response if invalid job
+  if (!job) {
+    return next(new ErrorResponse(`Job does not exist`, 400));
+  }
```

Comment:
can be shortened to
```suggestion
  // check if job exists
  const job = await db.one(
    'SELECT * FROM jobs WHERE job_id = $1',
    req.params.id
  );
```

Convention is for `404` response if resource does not exist, which this method will do, so no need for `400` later on

Reply:
Resolved.